### PR TITLE
Adding counters

### PR DESCRIPTION
**Pull request by davepmarshall**

### CODE REVIEW

File: server.js [hunk 1]
```diff
@@ -90,6 +91,8 @@ slapp.message('^(coffee|c|:coffee:)$',['ambient', 'mention'], (msg) => {
         if (teaUsers[channel].length != 0) {
           var teaMakerId = teaUsers[channel][Math.floor(Math.random()*teaUsers[channel].length)]
           var teaMaker = '<@' + teaMakerId + '>'
+          client.incr('makecount-' + channel + '-' + teaMakerId)
```
Comment (davepmarshall) [inline server.js]: Should this be teaMakerId or teaMaker ?

File: server.js [hunk 2]
```diff
@@ -191,6 +196,8 @@ slapp.message('^(tea|t|:tea:)$',['ambient', 'mention'], (msg) => {
...
```
Comment (davepmarshall) [inline server.js]: Should this be teaMakerId or teaMaker ?

**Comment by davepmarshall** [comment]
General comment, not sure if the redis incr command requires ```function(err, reply) ``` to work? @jordanpgl ?

File: server.js (hunk 1 above)
Comment (mradamfrancis) [reply server.js]: teaMakerId, so that we're consistent with drinkcount, startcount

**Review by mradamfrancis — CHANGES_REQUESTED** [review]
maybe we should do:
`client.incr('key1', function(err, reply) {
        console.log(reply); 
    });`
so that we check logs that it is incrementing correctly?

File: server.js (hunk 1 above)
Comment (davepmarshall) [reply server.js]: Cool

**Comment by davepmarshall** [comment]
Shall we ship it and see what happens?

**Pull request merged**